.travis.yml

```diff
@@ -1,11 +1,15 @@
-language: go
-go:
-  - 1.5.1
+language: python
+python:
+  - "2.7"
 branches:
   only:
     - master
 services:
   - docker
 sudo: required
+install:
+  - pip install -r rootfs/dev_requirements.txt
 script:
+  - make test
+after_success:
```

Comment:
I wouldn't move this to after success, since `after_success` cannot fail the build. It's intended for stuff like uploading build artifacts. Instead, I'd run it after the tests with `make test docker-build`.